jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -634,6 +662,57 @@ MlirToJlmConverter::ConvertFPSize(unsigned int size)
   }
 }
 
+llvm::linkage
+MlirToJlmConverter::ConvertLinkage(std::string stringValue)
```

Comment:
I would suggest this function takes a `std::string_view` instead, as it does not need its own owned string.